internal/batches/service/service.go

```diff
@@ -506,7 +506,9 @@ func validateMount(batchSpecDir string, spec *batcheslib.BatchSpec) error {
 	return nil
 }
 
-const exampleSpecTmpl = `name: NAME-OF-YOUR-BATCH-CHANGE
+const exampleSpecTmpl = `# "version" determines how the batch spec is interpreted.
+version: 2
+name: NAME-OF-YOUR-BATCH-CHANGE
```

Comment:
This is returned by `src batch new -f <file>`

Reply:
Tiny comment, as a user I would find `# "version" determines how the batch spec is interpreted` to be a bit mysterious. Maybe we could say
```
version: 2 # Use the latest schema version
name: ...
```